impl/src/util.rs

```diff
@@ -163,6 +163,10 @@ pub fn sanitize_ident(ident: &str) -> String {
   slug::slugify(&ident).replace("-", "_")
 }
 
+pub fn sanitize_name(name: &str) -> String {
```

Comment:
This is identical to sanitize_ident. Can that be used instead?

Reply:
This was made due to the suggestion at https://github.com/google/cargo-raze/pull/162#discussion_r433539678

Reply:
I guess lets hear if @dfreese intended to have a function with a different name but the same implementation. I think he expected that sanitize_name would just contain the "-" -> "_", but it ended up containing the "slugify" as well.

Reply:
The intention was for it to just be `name.replace("-", "_")`

Reply:
Though, it was in part because I wasn't completely familiar with slugify.  I guess it could work to just use the `sanitize_ident` function in all cases.  @acmcarther do you have a preference on that?

Reply:
Ah I see. For now, I have removed `sanitize_name` and changed it to call `sanitize_ident`. I can change it further if needed.